zask/ext/zerorpc/__init__.py

```diff
@@ -284,6 +284,7 @@ def server_after_exec(self, request_event, reply_event):
 
     def server_inspect_exception(self, request_event, reply_event, task_context, exc_infos):
         start = request_event.header.get('started_at')
+        request_time = lambda s: _milli_time() - s if s else 0
```

Comment:
Actually... no. I compacted that from an earlier long code and failed to realize `lambda()` is completely unnecessary. Let me quickly change that.